<launch>
    <arg name="web_mesh_path" default="/home/xiaoli/Dropbox/primp_exp/imagination" />
    <arg name="motion_primitive" default="scooping" />
    <arg name="object" default="red_bowl_1"/>
    <arg name="stomp_trajectory_topic" default="/server_stomp_planning/display_planned_path" />

    <param name="web_mesh_scene" value="$(arg web_mesh_path)/$(arg motion_primitive)/$(arg object)/perception_0/mesh_scene.obj" type="string"/>
    <param name="web_mesh_object" value="$(arg web_mesh_path)/$(arg motion_primitive)/$(arg object)/perception_0/mesh_obj.obj" type="string"/>
    <param name="repo_mesh_scene" value="$(find panda_moveit_control)/models/mesh_scene.obj" type="string" />
    <param name="repo_mesh_object" value="$(find panda_moveit_control)/models/mesh_obj.obj" type="string" />

    <param name="motion_primitive" value="$(arg motion_primitive)" type="string" />
    <param name="object" value="$(arg object)" type="string" />

    <param name="stomp_trajectory_topic" value="/server_stomp_planning/display_planned_path" type="string" />
    <param name="publish_stomp_trajectory_topic" value="/visualize_stomp_trajectory" type="string" />

    <node name="operate_planning_scene" pkg="panda_moveit_control" type="operate_planning_scene" output="screen" />


</launch>